poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
 
 [[package]]
 name = "alabaster"
 version = "0.7.13"
 description = "A configurable sidebar-enabled Sphinx theme"
-category = "dev"
```

Comment:
Yes, upgrading from Poetry 1.2.2 to 1.7.1 changes the lock format. 

NI internal usage of Poetry is finally upgrading to 1.7.1, so it would be reasonable to upgrade this repo's Github workflows.